src/main/java/ch/njol/skript/expressions/ExprSkull.java

```diff
@@ -22,7 +22,7 @@
 public class ExprSkull extends SimplePropertyExpression<OfflinePlayer, ItemType> {
 
 	static {
-		register(ExprSkull.class, ItemType.class, "(head|skull)", "offlineplayers");
+		register(ExprSkull.class, ItemType.class, "skull", "offlineplayers");
```

Comment:
```suggestion
		register(ExprSkull.class, ItemType.class, "(head [item]|skull)", "offlineplayers");
```
can't we just do this so users can specify that they want the head item in case they run into trouble with it? avoids breaking changes

Reply:
Judging by this PR and the other issue. The whole point was to make head not overlap with ExprEyeLocation and have ExprSkull work properly, and the only fix from past discussions was making head required in ExprSkull at least. Everyone is used to doing `head of player` as that is basic, over skull, usually. But then `head of player` used to return the location of player's head (where they are looking at), which isn't what people wanted. Maybe @Absolutionism has a better explaination? But that's just how I view it.

Reply:
then can't we just make it `head [location] of player`?

Reply:
oh lol, mb